backend/app/services/diagnosis_agent.py

```
@@ -323,20 +323,45 @@ async def fetch_weather(state: DiagnosisState) -> dict:
 
     # 2. 기상청 단기예보 조회 (약 3일치)
     from datetime import datetime, timedelta
-    current_time = datetime.now()
-    if current_time.hour < 2 or (current_time.hour == 2 and current_time.minute < 10):
-        target = current_time - timedelta(days=1)
-        target = target.replace(hour=23, minute=0, second=0)
-        base_date, base_time = target.strftime("%Y%m%d"), "2300"
-    else:
-        base_date, base_time = current_time.strftime("%Y%m%d"), "0200"
+    now_dt = datetime.now()
```

Ruff (0.15.12)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)